wemake_python_styleguide/violations/best_practices.py

```diff
@@ -865,7 +863,7 @@ class WrongKeywordViolation(ASTViolation):
     Forbids to use some ``python`` keywords.
 
     Reasoning:
-        Using some keywords generally gives you more pain that relieve.
+        Using some keywords generally gives you more pain than relieves.
```

Comment:
This sentence sounds slightly awkward. I'd suggest something like:
"Using these keywords is often more problematic than helpful."